.circleci/continue_config.yml

```diff
@@ -354,12 +354,12 @@ workflows:
           requires:
             - ci-base-<< matrix.python-version >>-<< matrix.extras >>
       - integration-test:
-          name: integration-test-<< matrix.pytest-group >>-<< matrix.python-version >>-<< matrix.extras >>
+          name: integration-test-workflow-<< matrix.python-version >>-<< matrix.extras >>
```

Comment:
this simplification makes the dynamic config generation obsolete, that step is not doing anything

Reply:
Nice to reduce complexity.

I think one thing missing is the conditional execution of integration tests in the updated version, i.e. only run dataset integration tests if `all=True` or `dataset= True`. I think it is possible by creating new job for workflow/dataset with conditional execution.